.travis.yml

```diff
@@ -8,9 +8,9 @@ services:
   - docker
 
 env:
-  - DOCKER_DB_VERSION=7.0.2
```

Comment:
why `7.0.2` is removed? what's the plan, having only one per Exasol major release?

Reply:
Yes. Latest per major. It is a trade-off between resource usage and coverage.